src/core/raster/qgscontrastenhancement.cpp

```diff
@@ -377,6 +377,54 @@ void QgsContrastEnhancement::readXml( const QDomElement &elem )
   }
 }
 
+void QgsContrastEnhancement::toSld( QDomDocument &doc, QDomElement &element ) const
+{
+  if ( doc.isNull() || element.isNull() )
+    return;
+
+  QString algName;
+  switch( contrastEnhancementAlgorithm() )
+  {
+    case StretchToMinimumMaximum:
+      algName = QStringLiteral( "StretchToMinimumMaximum" );
+      break;
+    /* TODO: check if ClipToZero => StretchAndClipToMinimumMaximum
+     * because value outside min/max ar considered as NoData instead of 0 */
+    case StretchAndClipToMinimumMaximum:
+      algName = QStringLiteral( "ClipToMinimumMaximum" );
+      break;
+    case ClipToMinimumMaximum:
+      algName = QStringLiteral( "ClipToMinimumMaximum" );
+      break;
+    case NoEnhancement:
+      return;
+    default:
+      QgsDebugMsgLevel( QStringLiteral( "No SLD1.0 convertion yet for stretch algorithm %1" ).arg( contrastEnhancementAlgorithmString( contrastEnhancementAlgorithm() ) ), 4 );
```

Comment:
I think this one could safely be QgsDebugMsg -- it's a valid error we'd always want to see, not a noisy debug message

Reply:
done